test/windows/event_log/logstash_event_log_plugin_integration.ps1

```diff
@@ -0,0 +1,137 @@
+# Created By: Gabriel Moskovicz
+# 
+# This is a script to test integration between logstash and elasticsearch.
+# It uses a simple json filter to parse the content of a simple text file an then
+# verifying if the message has been found in elasticsearch
+#
+# Requirements to run the test:
+# 
+# - Powershell 4
+# - Windows 7 or newer
+# - Java 7 or newer
+
+Add-Type -assembly "system.io.compression.filesystem"
+
+
+$Main_path = "C:\integration_test"
+If (Test-Path $Main_path){
+	ri -Recurse -Force $Main_path
+}
+$Download_path  = "$Main_path\download"
+md -Path $Download_path
+
+## Logstash variables
+
+$LS_CONFIG="test.conf"
+$LS_BRANCH="2.0"
```

Comment:
Same as the previous file https://github.com/elastic/logstash/pull/3994/files#r43676823

Allow people to override this in the environment but fallback on 2.0 if nothing is specified.